Future/DESCRIPTION

```diff
@@ -8,3 +8,7 @@ Description: Backend functions for shiny app Future.
 License: GPL-2
 Encoding: UTF-8
 LazyData: true
+Suggests: 
```

Comment:
Please bump version in 4, and add file to track changes e.g. `NEWS.md`

Reply:
done

Reply:
done